docs/src/examples/components/Popup/Variations/PopupExamplePosition.shorthand.tsx

```diff
@@ -1,54 +1,91 @@
 import * as React from 'react'
-import { Button, Grid, Popup } from '@stardust-ui/react'
+import { Button, Grid, Popup, Alignment, Position } from '@stardust-ui/react'
+import { useBooleanKnob, useSelectKnob } from '@stardust-ui/docs-components'
 
-const PopupWithButton = props => {
-  const { position, align, icon, padding } = props
+const PopupExamplePosition = () => {
+  const [open] = useBooleanKnob({ name: 'open shorthand', initialValue: true })
 
-  return (
-    <Popup
-      align={align}
-      position={position}
-      trigger={<Button icon={icon} styles={{ padding, height: '38px', minWidth: '64px' }} />}
-      content={{
-        content: (
-          <p>
-            The popup is rendered {position} the trigger
-            <br />
-            aligned to the {align}.
-          </p>
-        ),
-      }}
-    />
+  const [position] = useSelectKnob<Position>({
+    name: 'position shorthand',
```

Comment:
What does the word 'shorthand' mean in knob titles?

Reply:
I used it just to have a different name from the one in `PopupExamplePosition.tsx`. For knobs the name acts as an id so if I use the same name I get an error:

`Knob with name "${knob.name}" has been already registered`

Open to new name ideas that are more relevant

@miroslavstastny @layershifter

Reply:
Will check it...

Reply:
`packages/docs-components/src/knobs/KnobProvider.tsx` should be updated with this:

```jsx
  const registerKnob = (knob: KnobDefinition) => {
    setKnobs(prevKnobs => {
      if (process.env.NODE_ENV !== 'production') {
        if (prevKnobs[knob.name]) {
          throw new Error(`Knob with name "${knob.name}" has been already registered`)
        }
      }

      return { ...prevKnobs, [knob.name]: knob }
    })
  }
```

Reply:
@layershifter sent me a fix for this so I was able to use just the same name, fixed